### PR TITLE
feat: Added read-only card view [PT-184361154]

### PR DESCRIPTION
**Pull request by dougmartin**

### CODE REVIEW

File: src/components/card-view/card-view.scss [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+      justify-content: space-between;
+      align-items: center;
+      text-align: left;
+      margin: 0 0 10px;
+      padding: 3px;
+      border-radius: 4px;
+      border: solid 1px #cee9fb;
+      background-color: #fff;
+      font-weight: 600;
+      color: var(--text);
```
Comment (eireland) [inline src/components/card-view/card-view.scss:37]: Should we add this variable for the text color?

File: src/components/card-view/case-view.tsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+  const selectedCase = useMemo(() => cases[selectedCaseIndex], [selectedCaseIndex, cases]);
+
+  const style: React.CSSProperties = {backgroundColor: getBackgroundColor(level)};
+
+  useEffect(() => {
+    if (codapSelectedCase?.collection.id === collectionId) {
+      console.log("SELECTED", collectionId, selectedCase.collection.name, JSON.stringify(codapSelectedCase),);
+    }
+  }, [codapSelectedCase, selectedCase, collectionId]);
```
Comment (eireland) [inline src/components/card-view/case-view.tsx]: Do we still want this useEffect?

**Review by eireland — CHANGES_REQUESTED** [review]
Some clean up.

File: src/components/card-view/card-view.scss (hunk 1 above)
Comment (dougmartin) [reply src/components/card-view/card-view.scss]: No, I just removed the references: https://github.com/concord-consortium/multidata-plugin/pull/25/commits/9e19b24ab50ce45d944dfcc6d99ae38481d8dc21

**Review by eireland — APPROVED** [review]
👍 LGTM

**Pull request merged**